build/pkgs/sagelib/spkg-install

```diff
@@ -29,7 +29,7 @@ fi
 ## Building takes places in the build/ subdirectory.
 ##
 ## As a special exception, we feed SAGE_PKGS.
-## They are needed by src/sage/misc/package.py.  See meta-ticket #28815 for planned changes to this.
+## They are needed by src/sage/misc/package.py.  See github issue #28815 for planned changes to this.
```

Comment:
better as URL?

Reply:
I agree that it would probably be better as a URL, but there were enough of these changes that I didn't want to include that as part of this PR.